challenges/cat-photo-gallery/script.js

```diff
@@ -0,0 +1,41 @@
+const site = "https://cataas.com";
+const catImage = document.getElementById("cat-image");
+const enableTagCheckbox = document.getElementById("enable-tag");
+const tagInput = document.getElementById("tag-input");
+const loadButton = document.getElementById("load-button");
+
+async function loadRandomCat() {
+  try {
+    var url = generateUrl();
+
+    const response = await fetch(url);
+    if (!response.ok) {
+      throw new Error("Network response was not ok");
+    }
+
+    const data = await response.json();
+    const imageUrl = `${site}${data.url}`;
```

Comment:
this is not going to work.
the correct api links are:
- `https://cataas.com/cat?json=true` : for  a random cat
- `https://cataas.com/cat/:tag?json=true` : for a random tagged cat pic